tests/listener.ts

```diff
@@ -1,12 +1,12 @@
 import { deepStrictEqual } from "assert";
 import { listener } from "../src/listener";
-import { INTERNAL_SERVER_ERROR, OK } from "../src/status";
+import { HttpStatusCode } from "../src/status";
```

Comment:
If the purpose of having the module is just to have a namespace for status codes, you can always import like this, without having to add a custom class.

```ts

import * as HttpStatusCode from "../src/status"

```

Reply:
You're absolutely correct, and the only reasons I designed it this way were for potential future improvements/updates and to lessen the number of imports when using more than one status code in a single file, and because the class structure might allow for greater encapsulation and organization.

Reply:
I appreciate your effort here, and I understand your point about reducing the number of potential imports / exports in a file. Can this be solved in the same way that you cleaned up the exports in the `index.ts`?

Something like this maybe?

```ts
export * as HttpStatusCode from "./status";
```

I'm just cautious about using classes as they come with some more baggage, that in this particular case we can probably get away without.

What do you think?

Reply:
yes, you're right, with this type of export/import 
`export * as HttpStatusCode from "./status";`
The class doesn't seem to be necessary. since reducing import/export was my primary motivation for taking the class, and this appears to have fixed it.